### PR TITLE
Add new xAxis label position(BottomOutside) to HorizontalBarChart

### PR DESCRIPTION
**Pull request by anthony0926**
I add new xAxis label position that puts label on the top left of bar in HorizontalBarChart only.
screen shot 2016-01-27 at 4 35 41 pm

### CODE REVIEW

**Comment by danielgindi** [comment]
Thank you!
This is a nice addition.
Although there are a few problems:
1. All checks have failed. Which means tests are not passing (you can run them through Xcode), which probably means you've broken the rendering of something that you weren't supposed to...
2. There are unrelated changes in this PR, mainly a new podspec file...

**Comment by liuxuan30** [comment]
As the tests are about comparing images, so once the visual thing changes, the test would fail. such PRs would require to update the test images as well, right?

**Comment by danielgindi** [comment]
Yes but adding another position for the labels - should not make the screenshots different for another position...

**Comment by anthony0926** [comment]
Hi Daniel, 
I have added just few lines, so I couldn't understand why tests failed, 
Today I have some free time so I have checked your own code, FBScreenshotTestCase failed, 
So I think something was wrong before I added few lines.

Please see following screenshot,
screen shot 2016-01-29 at 8 59 18 pm

**Comment by danielgindi** [comment]
You are right!
Seems like the generated screenshots fell out of the commit for the 2.2.0 release. Not sure what happened there...
I've pushed them to master. Will review the PRs soon, has a few busy days :-)

File: Charts/Classes/Renderers/ChartXAxisRendererHorizontalBarChart.swift [hunk 1]
```diff
@@ -114,10 +124,8 @@ public class ChartXAxisRendererHorizontalBarChart: ChartXAxisRendererBarChart
             
             transformer.pointValueToPixel(&position)
             
-            if (viewPortHandler.isInBoundsY(position.y))
```
Comment (danielgindi) [inline Charts/Classes/Renderers/ChartXAxisRendererHorizontalBarChart.swift]: These changes seem unrelated to the PR

**Comment by danielgindi** [comment]
There are some changes in this PR which seem unrelated. The rest is good to go. Please clean it up and we'll merge

**Comment by jjatie** [comment]
This PR seems abandoned. Feel free to reopen if you are able to update it.
